ras_party/controllers/account_controller.py

```diff
@@ -190,6 +203,37 @@ def change_respondent_enrolment_status(payload, session):
                         desc='No active enrolments remaining for case')
 
 
+@with_db_session
+def disable_all_respondent_enrolments(respondent_email, session):
+    """Disables all enrolments for a respondent ,  returns the count of the removed enrolments"""
+
+    obfuscated_email = obfuscate_email(respondent_email)
+
+    logger.info('Disabling all enrolments for respondent', email=obfuscated_email)
+
+    removed_enrolments_count = 0
+
+    # raises errors if none or multiple, unusual import to avoid circular references
+    respondent = __import__('ras_party').controllers.respondent_controller.get_single_respondent_by_email(
```

Comment:
You'll have to explain this to me because this looks wrong even with the comment explaining why it's not wrong ^^

Reply:
We've talked and are going to move some code from the respondent controller into the account controller so we can remove this circular ref